webcam.go

```diff
@@ -106,6 +106,27 @@ func (w *Webcam) GetSupportedFrameSizes(f PixelFormat) []FrameSize {
 	return result
 }
 
+// GetSupportedFramerates returns supported frame rates for a given image format and frame size.
+func (w *Webcam) GetSupportedFramerates(fp PixelFormat, width uint32, height uint32) []FrameRate {
+	result := make([]FrameRate, 0)
```

Comment:
```suggestion
   var result []FrameRate
```

Reply:
Agree with these formatting changes. However, I was attempting to match the rest of the codebase.

How much do we care about consistency? Should we create a separate PR for formatting changes that would touch other parts?

Reply:
I think moving forward with cleaner patterns is good. If you have the time to PR the rest, sure!